uditagrawal08/task4.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        section{padding-top: 50px;
+            text-align: center;
+            margin-bottom: 500px;
+            margin-top: 50px;
+        }
+    </style>
+
+    <link rel="stylesheet" href="https://cc-bhu.github.io/web-development/assets/css/navbar.css">
```

Comment:
I found you successfully synced your branch with `cc-bhu`.
You can also use relative link for css that is `../assets/css/navbar.css`.